config.js

```diff
@@ -38,4 +38,13 @@ try {
 	//it is okay if no override file is provided
 }
 configInput.colors = getChalkColors(configInput, providedConfig);
+configInput._logLevels = {

```

Comment:
So I've seen log levels such as:

Bunyan logger:
Trace
Debug
Info
warn
error
fatal

and I've seen winston with:
debug
info
notice
warning
error
crit
alert
emerg

In that order...I have no reference point where to put our custom levels in the order, such as default, highlight and success...I more or less just picked something and went with it.

So keep in mind the order matter...if log level is set to success... only log statements of success or lower are logged.
if its set to default, it will only show default logs...